02LessonTwo/test/LessonTwoTest.js.orig

```diff
@@ -0,0 +1,43 @@
+var assert = require('assert');
```

Comment:
See if you can delete this file. You should be able to delete on your machine, the commit the change, and push it up. Then it should disappear from here.